packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -29,13 +31,35 @@ addGraphQLSchema(`
     contributors: [TagContributor!]
     totalCount: Int!
   }
+  type UserLikingTag {
+    _id: String!
+    displayName: String!
+  }
 `);
 
 export const TAG_POSTS_SORT_ORDER_OPTIONS: Record<string, SettingsOption>  = {
   relevance: { label: preferredHeadingCase('Most Relevant') },
   ...SORT_ORDER_OPTIONS,
 }
 
+// Define the helper function at an appropriate place in your file
+async function getTagMultiDocuments(
+  context: ResolverContext,
+  tagId: string,
+  projection = { baseScore: 1, extendedScore: 1 }
```

Comment:
The projection can't be configurable, since `getWithLoader` creates a new loader based on `loaderName` if one doesn't already exist, but calling it a second time with a different projection (or other options) would just cause it to ignore them.